tensorflow_serving/model_servers/main.cc

```diff
@@ -191,10 +193,16 @@ int main(int argc, char** argv) {
   tensorflow::string model_name = "default";
   tensorflow::int32 file_system_poll_wait_seconds = 1;
   tensorflow::string model_base_path;
+  tensorflow::int32 model_version_policy = FileSystemStoragePathSourceConfig::LATEST_VERSION;
   std::vector<tensorflow::Flag> flag_list = {
       tensorflow::Flag("port", &port, "port to listen on"),
       tensorflow::Flag("enable_batching", &enable_batching, "enable batching"),
       tensorflow::Flag("model_name", &model_name, "name of model"),
+      tensorflow::Flag("model_version_policy",
+                        &model_version_policy,
+                        "index for the version policy enum which determines the number of model versions to be served at the same time. "
```

Comment:
Please limit line length to 80 characters to adhere to the Google C++ style guide - https://google.github.io/styleguide/cppguide.html

You can also move &model_version_policy up to the line above it.